.github/workflows/release.yml

```diff
@@ -18,6 +18,43 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      # get k6 binaries macos
+      - name: get latest k6 binary macos
```

Comment:
Can we run the `install-k6` script here so that the code isn't duplicated?

Reply:
Great suggestion!
I built them in the opposite order so I didn't think about it, but if it works in the action and both locally for windows as well we might be able to just use the single script 👀

Reply:
I agree with running `node install-k6.js` instead to remove duplication.